### DISCUSSION TITLE
Windows build for v2?

### DISCUSSION THREAD

**Discussion by parasiteoflife** [Serene-Arc/bulk-downloader-for-reddit#291] [Q&A] [answered]
Please

**Comment by aliparlakci** [comment]
What do you exactly mean? This project is platform independent. You can use it on Windows also.

**Comment by parasiteoflife** [comment]
@aliparlakci Wont this project get any more builds like v1? I'm referring about getting an executable to just run or run through cmd/bat files.
Thanks.

**Comment by aliparlakci** [comment, marked as answer]
Actually, installation process has been changed but it didn't get any longer than before.

Now, you go to https://www.python.org/ and download the latest version there. Then, open a command prompt and run 
```
python -m pip install bdfr
```

That's all. You can now run the sample commands in the README as you would do with v1.

Also, this installation is global. That is, you can run your .bat files in any directory without having to have project files next to it.

**Comment by parasiteoflife** [comment]
That's great. I'm trying to install it but I got this:

```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
archivebox 0.5.4 requires requests==2.24.0, but you have requests 2.25.1 which is incompatible.
```

Will this be a problem?

**Reply by aliparlakci** [reply]
Try to run the program. I don't think there will be any issue.

**Reply by Serene-Arc** [reply]
Yes, that's a warning not an error.  It should be fine to run regardless.
